rush.json

```diff
@@ -369,6 +369,16 @@
         "projectFolder": "interfaces",
         "shouldPublish": true
     },
+    {
+        "packageName": "@binaris/shift-interfaces-node-client",
+        "projectFolder": "interfaces-client",
+        "shouldPublish": false
```

Comment:
Why not?

Reply:
I publish from `interfaces`.  Publishing from here would be odd, we would be using the *wrong* `package.json` -- the one in `gen/`, which (probably) means a different path from what we get this way.

Reply:
OK